meetings/agenda-minutes-2020-09-18.md

```diff
@@ -0,0 +1,289 @@
+# Sept 18, 2020
+
+## Attendees
+
+- Ujjwal Sharma
+- Philip Chimento
+- Philipp Dunkel
+- Shane F Carr
+- Justin Grant
+- Matt Johnson-Pint
+- Richard Gibson
+
+## Agenda
+
+* Is there any other feedback from the [feedback spreadsheet](https://docs.google.com/spreadsheets/d/10KsORWM1yBWej9sHbRbLLoGyYpFK2I71nveJkadF0LU/edit#gid=0) that we should consider incorporating? (30 min)
```

Comment:
Remove the link (and make sure it is not linked anywhere else)

Reply:
Would you prefer if I remove this and force push to get it out from the history?

Reply:
Removed. Force pushing.